.github/workflows/ci.yml

```diff
@@ -109,7 +109,7 @@ jobs:
 
       - name: Fetch sources for sibling projects
         run: |
-          git clone --depth=50 --branch=master https://github.com/SasView/sasdata.git ../sasdata
+          git clone --depth=50 --branch=release_0.9.0 https://github.com/SasView/sasdata.git ../sasdata
           git clone --depth=50 --branch=master https://github.com/SasView/sasmodels.git ../sasmodels
```

Comment:
When merging 6.0 back into main, should we not keep the sasdata branch being pulled in the ci.yml at sasdata/master? 6.0.0 should pull from release_0.9.0.

Checking sasdata/release_0.9.0 I note there have been no commits to it that are not in master so this should be safe?

Reply:
I agree with @butlerpd - this needs to point to the sasdata `master` branch before merging.